lib/project/pro_motion/activities/pm_activity.rb

```diff
@@ -71,6 +71,13 @@ def onOptionsItemSelected(item)
       super
     end
 
+    def onKeyDown(key_code, event)
+      if self.fragment
+        # share onKeyDown with super?
+        super if self.fragment.on_key_down(key_code, event)
```

Comment:
Do you think this behavior will be expected? The return value of `on_key_down` ? Just curious, no opinion.